packages/dashboard/src/components/composed/Contracts.tsx

```diff
@@ -0,0 +1,5 @@
+function Contracts(): JSX.Element {
+  return <div>interact with contracts</div>;
```

Comment:
If we're keeping this path maybe we can center it and change it to `Contract interactions coming soon` or something

Reply:
Though I know this path isn't advertised, so maybe we don't care

Reply:
Got rid of the route, so this isn't reachable anymore 🙈